app/javascript/chat/draw/index.jsx

```diff
@@ -0,0 +1,169 @@
+import { h } from 'preact';
+import { useRef, useEffect, useState } from 'preact/hooks';
+import PropTypes from 'prop-types';
+import { Button } from '@crayons';
+
+function Draw({ sendCanvasImage }) {
```

Comment:
Provide jsdoc documentation. For an example with a Preact component, see https://github.com/forem/forem/blob/master/app/javascript/utilities/dragAndDrop.js#L62-L79